election-guidelines-draft.md

```diff
@@ -0,0 +1,73 @@
+| 1st Monday of May| Election begins.     |
+| 2nd Monday of May| Election closes, and votes counted. Election results announced in the same week.|    
+| 3rd Monday of May| Previous Steering Committee to meet the newly elected Committee for official transition.| 
+| June 1 | New term begins with elected Steering Committee. Steering Committee Emeritus members help with the transition for the month of June.      |
+
+
+## Eligibility 
+
+### Eligibility for Steering Committee candadicy
+
+<code>**[TODO]** Candadicy eligbility needs to be discussed </code>
+
+Candidates can either be self-nominated or endorsed by one of Member Companies. A self-nominated candidate must at least be a contributor. 
+
+ 
+### Eligibility for voting
+
+To participate in the Steering commitee election, you must be a Contributor to the ONNX project. As defined in the community guideline, Contributor is sponsored by 2 approvers from different companies. 
+
+Contributors are further required to submit their github handle, email address, and affiliated company name to be eligible for voting. Any Contributor who has not submitted their information by before April 31st will not be able to participate in the election. 
+
+<code>**[TODO]** There needs to be a list of contributors</code> 
+
+## Candidacy process
+
+## Voting process 
+
+### General election procedure
+In order to promote fairness, the Steering Committee has decided to limit 1 vote per Member Company. Contributors will be able to vote individually, but their votes will be rolled up to represent the vote of associated Member Company. This procedure will prevent large companies with lots of Contributors from dominating the election results. 
+
+### Voting mechanics and algorithm
+
+The election will use [Condorcet ranking](https://en.wikipedia.org/wiki/Condorcet_method) with [Schulze method](https://en.wikipedia.org/wiki/Schulze_method). Condorcet ranking allows voters to indicate ranked preference for candidates, and Schultz method provides an algorithm to tally the overall preference.
+
+For ONNX Steering Committee election, the Condorcet ranking with Schulze method will be performed twice. The individual Contributor votes gets tallied first to Member Companies, and the results of the Member Company votes are ranked again using the same method. 
+
+### Voting platform
+We will use Condorcet Internet Voting Service ([civs.cs.cornell.edu](https://civs.cs.cornell.edu/)) to collect votes from Contributors. 
+
+After votes are casted, the results of individual votes will be uploaded to ONNX Github election directory to ensure transparency. 
+
+<code>**[TODO]** Determine if we can use other platforms</code> 
+
+## Election officers and Steering Committee emeritus members
+
+### Election officers
+Two election officers will be chosen from the current Steering committee to oversee the election process. 
+<code>**[TODO]** Confirm with Steering Committee </code> 
```

Comment:
what are their responsibilities?

Reply:
They would be responsible for overseeing the progress of the election and ensure the process is correctly implemented. For example, sending out an announcement email for election, organizing meetings for campaigns, etc. 

We could also have the whole Steering Committee to work on this together. Any thoughts?